lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb

```diff
@@ -74,6 +75,9 @@ def self.run(params)
         if xcode_project_file
           command += "--xcode-project-file #{xcode_project_file} "
         end
+        if use_hermes
+          command += "--use-hermes"
```

Comment:
```suggestion
          command += "--use-hermes "
```

Reply:
What if we wan't to bypass disabling hermes? I think it should be
```
command += "--use-hermes #{use_hermes} "
```

Reply:
you're right, it should be `--use-hermes #{use_hermes}` let me update

Reply:
added check and add some test for sure